src/system/SystemPacketBuffer.h

```diff
@@ -606,5 +641,18 @@ inline void PacketBuffer::SetDataLength(uint16_t aNewLen, const PacketBufferHand
     SetDataLength(aNewLen, aChainHead.mBuffer);
 }
 
+inline PacketBufferHandle PacketBuffer::Next()
+{
+    return PacketBufferHandle::Hold(Next_ForNow());
+}
+
+inline PacketBufferHandle PacketBuffer::Last()
+{
+    PacketBuffer * p = this;
+    while (p->next != nullptr)
```

Comment:
Should this use `NextForNow()` instead of poking members directly?

Reply:
Yes. `Next_ForNow()` will probably end up `private … NextBuffer()` or something like that once the public uses are gone. (The only one I don't have a queued PR for yet is TLV, which is a bit painful due to its OO‐in‐C‐with‐int‐as‐generic‐pointer style.)